tasks/set-payouts.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from 'fs';
+import { Signer } from '@ethersproject/abstract-signer';
+import { task } from 'hardhat/config';
+import { PayoutFields } from '../types';
+
+task('set-payouts', 'Sets a payout mapping for testing', async (_taskArgs, hre) => {
```

Comment:
How will we actually go about setting these payouts? Do we need a script to do that in production as well?

Reply:
Yep! That script will be written entirely in gitcoin/web since it's dependent on the CLR match objects in the DB